src/App.vue

```diff
@@ -81,15 +81,15 @@ export default {
 
 	mixins: [
 		talkHashCheck,
-		sessionIssueHandler,
 		isMobile,
 	],
 
 	setup() {
-		const isInCall = useIsInCall()
-		const supportSessionState = useActiveSession()
-
-		return { isInCall, supportSessionState }
+		return {
+			isInCall: useIsInCall(),
+			isLeavingAfterSessionIssue: useSessionIssueHandler(),
```

Comment:
`isLeavingAfterSessionIssue` is not used here

Reply:
I'd disagree here, i see two occurences of this ref:

https://github.com/nextcloud/spreed/blob/c67f8ad543a64d8ffb37c7b53bfe5b417c46b51c/src/App.vue#L122

https://github.com/nextcloud/spreed/blob/c67f8ad543a64d8ffb37c7b53bfe5b417c46b51c/src/App.vue#L296